proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -282,6 +282,12 @@ message Params {
   uint32 historical_entries = 4 [(gogoproto.moretags) = "yaml:\"historical_entries\""];
   // bond_denom defines the bondable coin denomination.
   string bond_denom = 5 [(gogoproto.moretags) = "yaml:\"bond_denom\""];
+  // min_commission_rate is the chain-wide minimum commission rate that a validator can charge their delegators
+  string min_commission_rate = 6 [
```

Comment:
This would be state breaking and needs a coordinated upgrade. If a chain were to have 1/3 upgraded only the chain would halt. I love the motivation, but we can't backport this